### ISSUE TITLE
I need your help :) debug mode

### ISSUE THREAD

**Issue by hyunhoJeon** [Phylliida/orbslam-windows#3]
As a result of executing your suggestion, the program works well in release mode and succeeded in running the stereo_kitti example. 

Then I set breakpoints to know how the program works specifically, but this does not work. 
Also, an error occurs when switching to debug mode in stereo_kitti example.

What should i do??

**Comment by Phylliida** [comment]
Did you rebuild all the dependencies also in debug mode?

**Comment by hyunhoJeon** [comment]
Thanks to your reply @Phylliida 

Yes, I built the dependencies except g2o in debug mode as you mentioned.

And one question is, do I just convert to debug mode and build it? 

Do I need to change anything else? For example, code generation ... 

I just changed to debug mode and tried to build it.  And error occurred ...

What should I do?? I look forward your reply. Thanks:)

**Comment by Phylliida** [comment]
Yea so you'll want to switch them to debug mode (in the little dropdown menu at the top of Visual Studio), then go into code generation and change it to Multi-threaded Debug (/MT) for each one

**Comment by hyunhoJeon** [comment]
Thank you so much @Phylliida 
I have seen cmakelist and saw that there is nothing related to debug. After that, I found out that I need to check the additional directory of the Stereo_kitti example and insert debug instead of release, and I have confirmed that the project works well. The rest of the process is exactly what you explained! Thank you so much! :)

+Sadly, I do not know why the runtime library error occurs during the debug process. Specifically, I get an error in the thread join function. Debugging multithreaded is difficult.
